### PR TITLE
Add envs for optimism deposits

### PR DESCRIPTION
**Pull request by sl1depengwyn**

### CODE REVIEW

File: for-developers/information-and-settings/env-variables.md [hunk 1]
```diff
@@ -164,6 +164,13 @@ This table is horizontally scrollable, version information is located in the las
...
+| `INDEXER_OPTIMISM_L1_DEPOSITS_START_BLOCK_L1`      |          | The number of the L1 block from which deposits will be fetched. Implemented in [#6993](https://github.com/blockscout/blockscout/pull/6993).                                                                                                                                                                                                                                                                                                                                                                                   | (empty)       | master  |
```
Comment (varasev) [inline for-developers/information-and-settings/env-variables.md]: This should be named as `INDEXER_OPTIMISM_L1_DEPOSITS_START_BLOCK` (without redundant `_L1` postfix).

Comment (varasev) [reply for-developers/information-and-settings/env-variables.md]: And `Optimism Rollup Management` should be added to the Table of Contents above like other sections

Comment (sl1depengwyn) [reply for-developers/information-and-settings/env-variables.md]: Done
Note that it seems that we do not have `Management` for all other sections of table of contents, so I removed it, wdyt?

Comment (varasev) [reply for-developers/information-and-settings/env-variables.md]: Ok

**Pull request merged**